lib/productions/extended-attributes.js

```diff
@@ -75,6 +77,13 @@ class SimpleExtendedAttribute extends Base {
   }
 
   *validate(defs) {
+    if (this.name === "NoInterfaceObject") {
+      const message = `\`[NoInterfaceObject]\` extended attribute is an \
+undesirable feature that can be removed in the future. Refer to the \
```

Comment:
```suggestion
undesirable feature that may be removed from WebIDL in the future. Refer to the \
```

Reply:
Maybe "Web IDL" because that's the official title?

Reply:
Done